ginivision/src/main/java/net/gini/android/vision/camera/CameraActivity.java

```diff
@@ -297,7 +297,8 @@
  * </ul>
  * </p>
  **/
-public class CameraActivity extends AppCompatActivity implements CameraFragmentListener {
+public class CameraActivity extends AppCompatActivity implements CameraFragmentListener,
+        CameraFragmentInterface {
```

Comment:
I'm not really sure about this interface name. When I have seen it, I have thought about a generic java `interface`, not as an interface related to user interface.

Reply:
It is a generic interface and it contains only UI related methods now which might change in the future.